plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/PageValidator.java

```diff
@@ -31,11 +31,17 @@ public PageValidator(AppEngineWizardPage wizardPage) {
 
   @Override
   public void modifyText(ModifyEvent event) {
-    wizardPage.setPageComplete(wizardPage.validatePage());
+    run();
```

Comment:
Why Runnable if we're just invoking directly? Does some other code execute this in a thread?